generator/src/test/java/com/scottlogic/deg/generator/cucumber/utils/RowsPresentMatcher.java

```diff
@@ -21,44 +19,41 @@ public RowsPresentMatcher(List<List<Object>> expectedRows) {
 
     @Override
     public boolean matches(Object o) {
-        List<List<Objects>> actualRows = (List<List<Objects>>) o;
+        List<List<Object>> actualRows = (List<List<Object>>) o;
         return getMissingRowMatchers(actualRows).isEmpty();
     }
 
     @Override
     public void describeTo(Description description) {
         description.appendText(
-            String.join(
-                ", ",
-                getExpectedMatchers()
-                    .stream()
-                    .map(matcher -> matcher.toString())
-                    .collect(Collectors.toList())));
+            getExpectedMatchers()
+                .stream()
+                .map(BaseMatcher::toString)
+                .collect(Collectors.joining(", ")));
     }
 
     @Override
     public void describeMismatch(Object item, Description description) {
-        List<List<Objects>> actualRows = (List<List<Objects>>) item;
```

Comment:
Have we got any other usages of `Objects` we need to swap in with `Object` and is this worth highlighting for future PRs. From my short google the java.util.Objects is a static class that makes no sense here?

Reply:
A quick look shows that this PR contains all the switches to `Object` from `Objects`